.gitignore

```diff
@@ -6,6 +6,7 @@
 # Ignore all files and folders in root
 *
 !/conda-forge.yml
+!abs.yaml
```

Comment:
```suggestion
!/abs.yaml
```

in this way we consider only the `abs.yaml` in the root folder, and we avoid mistakes to add the file into the `recipe` folder.

Reply:
Nice catch ! I corrected as suggested.